go.mod

```diff
@@ -359,7 +359,7 @@ replace (
 	github.com/naoina/go-stringutil => github.com/naoina/go-stringutil v0.1.0
 	github.com/naoina/toml => github.com/naoina/toml v0.1.1
 	github.com/natefinch/lumberjack => github.com/natefinch/lumberjack v2.0.0+incompatible
-	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.6.0
```

Comment:
Any reason to bump this if it's already a tagged release?

Reply:
One of the other dependencies needed this update @andrewsykim

Reply:
ah okay, thank you